qt/widgets/spectrumviewer/src/SpectrumView.cpp

```diff
@@ -154,6 +155,19 @@ void SpectrumView::renderWorkspace(
   m_ui->imageTabs->setCurrentIndex(tab);
 }
 
+/**
+ * Renders a new workspace on the spectrum viewer.
+ *
```

Comment:
Should explain what will happen if a workspace with this name does not exist. Might also extend this to explain how the workspace will be rendered. Different tabs etc? Can be fixed as part of the follow up PR that @MikeHart85  is working on.

Might also have been nice (but not essential) to have been able to pass in some iterable to the spectrum viewer constructor with all the workspace names to view.